coral-erm/coral
# (issue #388) German translation

This patch adds a German translation to Coral and adds German as an option to the language pull down.

Test plan:
 - Apply the patch
 - Make sure you have the `de_DE.UTF-8 UTF-8` locale installed on your system (`sudo dpkg-reconfigure locales` on Debian-based systems)
 - Restart PHP and your webserver
 - German should be available in the language dropdown list, and Coral should switch to german when asked for.
